pkg/features/kube_features.go

```diff
@@ -598,7 +599,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	TTLAfterFinished:               {Default: false, PreRelease: featuregate.Alpha},
 	KubeletPodResources:            {Default: true, PreRelease: featuregate.Beta},
 	WindowsGMSA:                    {Default: true, PreRelease: featuregate.Beta},
-	WindowsRunAsUserName:           {Default: true, PreRelease: featuregate.Beta},
+	WindowsRunAsUserName:           {Default: true, PreRelease: featuregate.GA, LockToDefault: true},
```

Comment:
Should the `LockToDefault` be accompanied with a comment on which k/k version will be suitable to remove the flag completely? E.g. in https://github.com/kubernetes/kubernetes/pull/86210/files

Reply:
I see a few features that have LockToDefault: true and no comment but i'll go ahead and add one for 1.20 anyways.
I'll rebase oncet hose changes are done.